gradle/validation/forbidden-apis.gradle

```diff
@@ -99,7 +99,6 @@ allprojects { prj ->
           "**.SuppressForbidden"
       ]
 
-      doFirst dynamicSignatures.curry(configurations.toolsCompileClasspath, "lucene")
```

Comment:
I don't quite understand this line but the reference to "lucene" probably matches nothing.